api/modules/dataSyncs.js

```diff
@@ -12,6 +13,45 @@ const { GITHUB, TOGGL } = require('../config/credentials')
 
 const dataSyncs = module.exports = (() => {
 
+    const importInvoicelyCsvToStripe = async () => {
+        const invoiceFile = INVOICELY_CSV_PATH
+        const csvFile =
+            await amazon.fetchFile({ file: invoiceFile })
+                .then(file => {
+                    return invoicelyCodebase.modelCSV(file)
+                })
+                .catch(err => {
+                    console.log('error', err);
+                    return err.message
+                })
```

Comment:
There is no need to use `await` together with `.then/catch` you can just do:

```
const csvFile = await amazon.fetchFile({ file: invoiceFile })
const modeledCsv = await invoicelyCodebase.modelCSV(csvFile)
```

If you need to catch an error use a try/catch or move this into the try/catch you already wrote below

```
const stripeCustomers = await stripe.listAllCustomers()
const customersFromCsv = []
try {
    const csvFile = await amazon.fetchFile({ file: invoiceFile })
    const modeledCsv = await invoicelyCodebase.modelCSV(csvFile)
    modeledCsv.map(async csvData => {
        ...
} catch (err) { }
```